Formula/leela-zero.rb

```diff
@@ -0,0 +1,42 @@
+class LeelaZero < Formula
+  desc "Neural Network Go engine with no human-provided knowledge"
+  homepage "https://zero.sjeng.org/"
+  # pull from git tag to get submodules
+  url "https://github.com/leela-zero/leela-zero.git",
+      :tag      => "v0.16",
+      :revision => "4fd6e694ead45b7cbb14ea76c5bdfcbfa662304b"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  resource "network" do
+    url "https://zero.sjeng.org/networks/2d46c3273204ba8e41bc2daffd61c42da275401321f4f7a62e45f26472b7c8c2.gz", :using => :nounzip
```

Comment:
This is akin to shipping unversioned data along with the formula. It requires a special parameter to use, and users will probably want to update it anyway.

How about shipping only the software? After all, if the network is not provided in the source download, upstream doesn't consider it part of the tool itself.

Reply:
It's not unversioned - I specifically used a tagged version to address the unversioned argument :) You can see all the tagged networks at https://zero.sjeng.org/network-profiles

It's also not optional; the `-w` "special parameter" is mandatory and Leela Zero does absolutely nothing without it. Leela Zero is, at its core, a neural network engine, and the network file is the network it uses. Yes, it occasionally gets upgraded as more training occurs, but most users won't need a new version very often.

Reply:
If it's useful, http://zero.sjeng.org/best-network is always the best available network.